source/Node.js

```diff
@@ -118,6 +160,25 @@ function isContainer ( node ) {
     return ( type === ELEMENT_NODE || type === DOCUMENT_FRAGMENT_NODE ) &&
         !isInline( node ) && !isBlock( node );
 }
+
+// // neilj1.8.9
```

Comment:
I didn't want to switch to this right away even though there is mention of performance benefits because there could be some subtle bugs here.